src/types/collaborative/session.ts

```diff
@@ -7,6 +8,11 @@ export interface Client {
   id: ClientId;
   name: string;
   position?: ClientPosition;
+  color?: Color;
```

Comment:
I'm not sure why `color?` is needed here ? Everything builds without it.
If you remove it, `ClientWithPosition` can also be removed (and revert associated changes)